homework_5/homework_5.py

```diff
@@ -0,0 +1,18 @@
+a = input("enter the text: ")
+#number is even or odd
+if a.isdigit() is True:
```

Comment:
Your program should work with every symbol in the input string. add here `for symbol in a` to iterate through all symbols